### PR TITLE
fix: fixed badge component

### PR DESCRIPTION
**Pull request by partha8**
Hey everyone, while reviewing my code, I noticed that I made a blunder when I created the badge component. Here is the updated component. Please review the code and approve.

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -53,7 +50,22 @@
   background: var(--clr-grey-80);
```
Comment (vrx29) [inline components/badge/badge.css:50]: just a suggestion naming variables in caps would improve readability. Everything else looks good

Comment (partha8) [reply components/badge/badge.css:50]: I understand, but it depends on the preferences of the developer. For readability, I make sure to write "clr" followed by the color name, which is followed by a number to let another developer know how dark or light the hue is. I even asked Animesh about this and he approves. Thank you so much for reviewing @vrx29.

**Comment by anik31** [comment]
Looks good to me

**Comment by partha8** [comment]
Got two reviews for the fix, merging it to the dev branch.

**Pull request merged**